docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-designing-a-case-proc.adoc

```diff
@@ -50,7 +50,14 @@ image::case-variables.png[Editor for IT Orders variables]
 +
 In the IT Orders sample project, you can see that certain variables are identified as `Case File`. A case file variable is stored in the case file itself and promotes reuse of the case file data rather than copying it across multiple process instances. For example, the `hwSpec` variable is available to all processes within the case and is accessible directly from the case file even without an existing process instance. 
 +
-For more information about the case file, see <<case-management-case-file-con-{context}>>.
+For more information about the case file, see 
+ifeval::["{context}" == "case-management-getting-started"]
+<<case-management-case-file-con-case-management-getting-started>>
+endif::[]
+ifeval::["{context}" == "case-management-design"]
+<<case-management-case-file-con-case-management-design>>
+endif::[]
+.
```

Comment:
General comment about cross-refs throughout the doc.

The majority of the time you use the ref format "<<anchor-id-context-A>>", "<<anchor-id-context-B>>", etc., and occasionally "xref:anchor-id-context-A", etc. We decided back when we started modularizing that we would start using only the xref: format (Contrib Guide) because the << >> format starts to break down in certain cases of cross-referencing reused modules. I'm surprised you were able to use the ifevals using the << >> way! I found that it only worked with xref and hit issues with the << >> when referencing the same module in multiple places. So that's cool if it works. There was one other reason though that I can't recall. It's up to you I guess. So long as it works the same, I suppose it's all that really matters.

Also, this is super, SUPER trivial and I honestly don't care if you heed it or not, just informing. The actual format for setting anchor IDs with a context is to use hyphens all around but underscore before the context attribute: "[id='anchor-id_{context}']". This would make the ref the following, for example: `xref:case-management-case-file-con_case-management-design`. Makes zero difference to the function, but makes it easier to delimit the anchor ID from the context variable in cross-refs, in the rendered doc's URL, etc., which is why we went with that. Section about this in the contrib guide: http://ccs-jenkins.gsslab.brq.redhat.com:8080/view/Topic%20branches/job/doc-Red_Hat_JBoss_BxMS_Contribution_Guide-branch-shared-repo/lastSuccessfulBuild/artifact/index.html#reusing-modules

Again, completely up to you if you want to ignore this. I know it's one of many, many things around here that are error prone and arbitrary. Just wanted to at least bring it up.

Reply:
I'm not sure when the discussion around <> vs xref tags happened, but I started doing the <<>> ones when we were putting out LA because I saw that's how everyone else had been doing internal links. I've been doing them that way ever since. 

I'm not able to change all of those links at this point and as you said, if it's working then that's all that matters for now. If possible, it'd be good to have changes communicated through email or something because I don't check the Contributor Guide unless I'm looking for a specific thing. I am easily left out of the loop having no overlap with other writers and sometimes these things don't get picked up by anyone during peer reviews so I just keep doing them until told otherwise.

Reply:
@gsheldon, yeah, the time diff is killer on communications and other stuff. Sorry about that. There were several emails around the xref change and file/anchor naming though that I circulated back in about Sept/Oct of last year, one set of emails taking team input, and another confirming the decision. The only reason I remember it was that was about the time Marek told me not to expect too many writers to respond :) He said usually people don't say anything unless they care. But sometimes it could actually be missed, like in this case.

Either way, agreed that it's not worth your trouble to "fix" in retrospect. So long as it works, that's great.